api/client/options.proto

```diff
@@ -14,6 +14,49 @@ message RequestOptions {
   google.protobuf.UInt32Value request_body_size = 3 [(validate.rules).uint32 = {lte: 4194304}];
 }
 
+// We wrap all values so we can have a unified way of option handling with respect to
+// defaults, merging, etc. As there's no stock concept for enumerations, we manually
+// define custom wrappers for them. These used to be strings, which did provide the
+// wrapped type.
+message AddressFamily {
+  enum AddressFamilyOptions {
+    AUTO = 0;
+    V4 = 1;
+    V6 = 2;
+  }
+  AddressFamilyOptions value = 1;
+}
+
+message Verbosity {
+  enum VerbosityOptions {
+    INFO = 0;
```

Comment:
I think you want this and the others below to have an `UNSET` or `DEFAULT` value, since the currently assigned values are a policy decision.

Reply:
Right, let me amend that

Reply:
Done